docs/reference/release-notes/6.3.asciidoc

```diff
@@ -98,11 +98,29 @@ elasticsearch plugin.
 //[float]
 //=== New Features
 
-//[float]
-//=== Enhancements
+[float]
+=== Enhancements
 
-//[float]
-//=== Bug Fixes
+Machine Learning::
+* Synchronize long and short tests for periodicity {ml-pull}62[#62]
+* Improvements to trend modelling and periodicity testing for forecasting {ml-pull}7[#7] (issue: {ml-issue}5[#5])
+
+[float]
+=== Bug Fixes
+
+Machine Learning:: 
+
+* By-fields should respect model_plot_config.terms {ml-pull}86[#86] (issue: {ml-issue}30004[#30004])
```

Comment:
I think `ml-issue` should be simply `issue` on this line.

Reply:
Good catch, thanks!